### PR TITLE
fix startActivity

### PR DESCRIPTION
**Pull request by Pmant**

### CODE REVIEW

**Comment by Pmant** [comment]
#7

File: lib/harmonyclient.js [hunk 1]
```diff
@@ -92,34 +92,25 @@ function getActivities() {
...
-				canHandleStanza = true;
-			}
-		}
-
-		return canHandleStanza;
-	});
-}
+ function startActivity(activityId) {
+    var timestamp_temp = Date.now() - timestamp;
+    var body = 'activityId=' + activityId + ':timestamp=' + timestamp_temp;
```
Comment (swissmanu) [inline lib/harmonyclient.js]: what is this timestamp calculation about?

**Comment by swissmanu** [comment]
the code looks good. would you mind to preserve tab- and whitespace style from the original code?

**Comment by Pmant** [comment]
fixed

**Comment by swissmanu** [comment]
thanks one more time @Pmant :)

**Comment by swissmanu** [comment]
included in release 1.1.5

**Pull request merged**